make/modules/java.desktop/lib/AwtLibraries.gmk

```diff
@@ -278,7 +278,6 @@ ifeq ($(call isTargetOs, windows macosx)+$(ENABLE_HEADLESS_ONLY), false+false)
       DISABLED_WARNINGS_gcc_X11TextRenderer_md.c := unused-but-set-variable, \
       DISABLED_WARNINGS_gcc_XlibWrapper.c := type-limits pointer-to-int-cast, \
       DISABLED_WARNINGS_gcc_XRBackendNative.c := maybe-uninitialized, \
-      DISABLED_WARNINGS_gcc_XToolkit.c := unused-result, \
```

Comment:
Probably a question for build team -
One thing to be noted here is that:  With unused-result line removed and WITHOUT the if block changes in XToolkit.c, I expected the build to fail due to unused-result on linux/gcc but it did not.
So is there any chance that this disabled warning is being added from else where for gcc ?

Reply:
Maybe gcc doesn't detect this specific case.

There could've been other cases which triggered the warning in gcc.

Reply:
It could be that not all versions of GCC trigger this warning in this particular case. We don't track disabling of warnings on a per version granularity, as that would be incredibly tedious, so it's hard to know. It could also be that this disabling of `unused-result` was added for a different case that has since disappeared.

Regardless, since you are fixing such an issue in the code and no longer see this warning with a reasonably new GCC, it should be safe to remove this.